app/routes.py

```diff
@@ -75,6 +75,10 @@ async def download_image(
     adjusted_x = int(x)
     adjusted_y = int(y)
 
+    # Apply any necessary adjustments to the coordinates
+    # For example, if the y-coordinate is off, adjust it here
+    adjusted_y += 10  # Adjust the y-coordinate by 10 pixels (example adjustment)
```

Comment:
This is not the right solution. I don't know what is going on here.

Let's add some logging to the FE and BE to figure out why the text seems to be off. The position of the text should be relative to the image.

Reply:
Sure, I understand that adding a fixed adjustment is not the right solution. Let's add some logging to both the frontend and backend to better understand why the text position is off. This will help us identify the root cause of the issue.

I'll update the PR to include logging statements that capture the coordinates and dimensions of the text and image elements both in the frontend and backend. This should give us more insight into what's going wrong.

I'll make the necessary changes and update the PR shortly. Thank you for the feedback!